web/src/server_events_dispatch.js

```diff
@@ -779,6 +780,9 @@ export function dispatch_normal_event(event) {
                 stream_list.update_streams_sidebar();
                 stream_list_sort.set_filter_out_inactives();
             }
+            if (event.property === "web_animate_image_previews") {
+                message_lists.current?.rerender();
+            }
```

Comment:
I suspect you need to do this using `all_rendered_message_lists`, to make sure the "combined feed" gets rerendered if it is cached. .

Reply:
updated!